acceptance/tests/options/custom_facts.rb

```diff
@@ -29,9 +35,6 @@
   create_remote_file(agent, custom_fact, content)
   on(agent, "chmod +x '#{custom_fact}'")
```

Comment:
Do custom facts need to be executable? We're not running them, we're loading them in Ruby.